automation/tests-container-utils.sh

```diff
@@ -78,7 +77,12 @@ function container_pre_test_setup {
     fi
 
     create_container
-    if [[ -v customize_cmd ]];then
+    if [[ -v "$customize_cmd" ]];then
+        if [[ "$CI" == "true" ]];then
+            container_exec "rm -fv /var/cache/dnf/metadata_lock.pid"
+            container_exec "dnf clean all"
+            container_exec "dnf makecache || :"
+        fi
```

Comment:
Can you wrap this into a function so that it could be used in `upgrade_nm_from_copr()` also?

Reply:
Done, thanks!